tests/r/FormatRBearTest.py

```diff
@@ -11,14 +10,108 @@
 }""".splitlines(keepends=True)
 
 
-bad_file = """1+1
+good_file_2 = """1 + 1
+if (TRUE) {
+    x <- 1  # inline comments
+} else {
+    x <- 2
+    print("Oh no... ask the right bracket to go away!")
+}""".splitlines(keepends=True)
+
+
+good_file_3 = """1 + 1
+if (TRUE) {
+    x = 1
+} else {
+    x = 2
+    print("Oh no... ask the right bracket to go away!")
+}""".splitlines(keepends=True)
+
+
+good_file_4 = """1 + 1
+if (TRUE)
+{
+    x = 1  # inline comments
+} else
+{
+    x = 2
+    print("Oh no... ask the right bracket to go away!")
+}""".splitlines(keepends=True)
+
+good_file_5 = """1 + 1
+if (TRUE) {
+        x = 1  # inline comments
+} else {
+        x = 2
+        print("Oh no... ask the right bracket to go away!")
+}""".splitlines(keepends=True)
+
```

Comment:
i add to remove the 'FormatRBearRWidthcutoffTest' because of this.
the space at the end is considered by formatR, hence by running coala the code fails